packages/components/nodes/documentloaders/Unstructured/Unstructured.ts

```diff
@@ -67,6 +69,7 @@ export class UnstructuredLoader extends BaseDocumentLoader {
         const options = optionsOrLegacyFilePath
         this.apiKey = options.apiKey
         this.apiUrl = options.apiUrl ?? this.apiUrl
+        this.apiHeader = options.apiHeader ?? 'UNSTRUCTURED-API-KEY'
```

Comment:
where does `options.apiHeader` comes from? Don't you need that value from `UnstructuredFile.ts` and `UnstructuredFolder.ts`? And you if we were to make this user-configurable, we have to add a new input there as well